pkg/controllers/common/services.go

```diff
@@ -0,0 +1,11 @@
+package common
```

Comment:
I suggest that we move this service interface definiation to  nsx/service/common module, so controller module don't need have any service related code.